### DISCUSSION TITLE
open -a Emacs no longer works on macOS 15.1

### DISCUSSION THREAD

**Discussion by jchonig** [d12frosted/homebrew-emacs-plus#741] [General]
I upgraded a 15.0 Intel MBP to 15.1 and now when I run `open -a Emacs`, I get:

```
$ open -a Emacs
_LSOpenURLsWithCompletionHandler() failed for the application /usr/local/Cellar/emacs-plus@30/30.0.92/Emacs.app with error -54.
```

I initially had emacs-plus@29 installed, but upgraded to emacs-plus@30 to see if that worked.

Emacs.app is linked to the brew install:

```
jch@jch-imbp[496]:Downloads$ l /Applications/Emacs.app
37819464 lrwxr-x---@ 1 admin  38 Oct 28 17:55 /Applications/Emacs.app@ -> /usr/local/opt/emacs-plus@30/Emacs.app
```

Opening `/usr/local/bin/emacs` directly works, with our w/o `-nw`.

Is this an emacs-plus error?  If not, any ideas?

**Comment by jchonig** [comment]
Another clue:

```
$ osascript -e 'tell application "Emacs" to launch'
28:34: execution error: File permission error. (-54)
```

**Comment by d12frosted** [comment]
Did you try running `brew doctor`?

**Comment by jchonig** [comment]
Yes, everything is healthy.

Although I'm in the US so I had a $20 copay.  :-(

**Comment by d12frosted** [comment]
Seems like it's a common issue https://github.com/d12frosted/homebrew-emacs-plus/issues/742
